packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -23,6 +23,16 @@ const double _kCupertinoFocusColorOpacity = 0.80;
 const double _kCupertinoFocusColorBrightness = 0.69;
 const double _kCupertinoFocusColorSaturation = 0.835;
 
+// Obtained from Apple's Design Resources (macOS Sonoma) figma template:
+// https://www.figma.com/design/vmIRIt4jgAmSFvXO0SAxU2/Apple-Design-Resources---macOS-(Community)
+const Color _kDisabledCheckColor = Color.fromARGB(255, 172, 172, 172);
+
+// Eyeballed from a checkbox on a physical Macbook Pro running macOS version 14.5.
+const CupertinoDynamicColor _kDefaultFillColor = CupertinoDynamicColor.withBrightness(
+    color: CupertinoColors.white,
+    darkColor: Color.fromARGB(255, 87, 87, 87),
```

Comment:
```suggestion
  color: CupertinoColors.white,
  darkColor: Color.fromARGB(255, 87, 87, 87),
```

Reply:
Also according to [my computation](https://github.com/flutter/flutter/pull/148804#issuecomment-2192600418) the dark color should not be an opaque color, but must be a gradient from a translucent white to another translucent white, otherwise it will look differently with non-black background.